tracker/src/plausible.js

```diff
@@ -364,6 +364,20 @@
     })
   }
 
+if (COMPILE_CONFIG && config.trackFormSubmissions) {
+  function trackFormSubmission(e) {
+    if (e.target.hasAttribute('novalidate') || e.target.checkValidity()) {
+      plausible('WP Form Completions', { props: { path: location.pathname } });
```

Comment:
The event name is incorrect. Don't worry about wordpress in the current stage, we'll figure out a way to deal with their legacy events/goals later.

```suggestion
      plausible('Form submission', { props: { path: location.pathname } });
```

Reply:
Went with 'Form Submission' because it's more inline with 'File Download' etc, exact name TBD.